server/statistics/store.go

```diff
@@ -87,6 +89,36 @@ func (s *StoresStats) ObserveRegionsStats(storeIDs []uint64, writeBytesRates, wr
 	}
 }
 
+// RemoveSlowStoreStatus removes SlowStoreStats with a given store ID.
+func (s *StoresStats) RemoveSlowStoreStatus(storeID uint64) {
+	s.Lock()
+	defer s.Unlock()
+	delete(s.slowStoresStats, storeID)
+}
+
+// ObserveSlowStoreStatus updates SlowStoreStats with a given store ID.
+func (s *StoresStats) ObserveSlowStoreStatus(storeID uint64, isSlow bool) {
```

Comment:
```suggestion
func (s *StoresStats) ObserveSlowStoreStatus(storeID uint64, status bool) {
```

Reply:
IMO, `isSlow` is more clear to represent `the given store is slow or not`.